ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -1129,14 +1129,17 @@ public void createTable(Table tbl, boolean ifNotExists,
           principalPrivs.setRolePrivileges(grants.getRoleGrants());
           tTbl.setPrivileges(principalPrivs);
         }
+        if (HiveConf.getBoolVar(conf, ConfVars.HIVE_TXN_LOCKLESS_READS_ENABLED) && AcidUtils.isTransactionalTable(tbl)) {
```

Comment:
i would consider do this for every transactional table, regarding of the config. So we would have consistent naming convention everywhere + it would help I think it would help reduce the cases to consider (like what happens if the config is on for some time, then it is turn off, or if somebody turns this off in session level etc.)

Reply:
i don't think it would be a problem, but i'll double check. Thing is that I am indirectly passing HIVE_TXN_LOCKLESS_READS_ENABLED config value to HMS via the txnId attribute. If set - HIVE_TXN_LOCKLESS_READS_ENABLED was enabled.